code/datums/status_effects/debuffs.dm

```diff
@@ -895,6 +895,11 @@
 						list("<span class='danger'>Your skin feels loose.</span>", "You feel very strange.", "<span class='danger'>You feel a stabbing pain in your head!</span>", "<span class='danger'>Your stomach churns.</span>"),
 						list("<span class='danger'>Your entire body vibrates.</span>")
 			)
+			fake_emote = list(
+						list(),
+						list(),
+						list()
+			)
```

Comment:
No actual fake emotes though?

Reply:
This one is supposed to have no emotes to replicate the "retro virus", which has no emote symptoms.

Reply:
Thought the retrovirus had emotes, my bad then